all/guides/wix-groups/docs/Flow.md

```diff
@@ -73,20 +73,20 @@ For example, create a report including aggregated group info: total amount of gr
 
-Your app can identify group members who are interested in a certain topic by searching for keywords in their answers to the membership questions.
-Then, your app could send emails to the identified members informing them about upcoming events or special offers.
+Your can identify group members who are interested in a certain topic by searching for keywords in their answers to the membership questions.
+Then, you can send emails to the identified members informing them about upcoming events or special offers.
 
-1. Add membership questions by calling the [Create or Replace All Membership Questions](https://dev.wix.com/docs/rest/crm/community/groups/membership-questions/create-or-replace-all-membership-questions) endpoint.
-2. At any time you want to send emails to selected members, call the [List Answers](https://dev.wix.com/docs/rest/crm/community/groups/membership-questions/list-answers) endpoint.
+1. Add membership questions by calling [Create or Replace All Membership Questions](https://dev.wix.com/docs/rest/crm/community/groups/membership-questions/create-or-replace-all-membership-questions).
+2. At any time you want to send emails to selected members, call [List Answers](https://dev.wix.com/docs/rest/crm/community/groups/membership-questions/list-answers).
 3. Identify the relevant members via a keyword search.
-4. Call the [Query Email Subscriptions](https://dev.wix.com/docs/rest/crm/communication/email-subscriptions/query-email-subscriptions) endpoint to confirm that the identified members have agreed to receive emails from the site owners.
+4. Call [Query Email Subscriptions](https://dev.wix.com/docs/rest/crm/communication/email-subscriptions/query-email-subscriptions) to confirm that the identified members have agreed to receive emails from the site owners.
 5. Prepare and send the automated emails.
 
 ## Bulk Update Group Rules
 
-The site onwers have decided to manually update the rules for one group. Your app can apply the new rules to other relevant groups.
+When Wix users manually update the rules for one group, You can apply the new rules to other relevant groups.
 
-1. Get the updated group rules by calling the [List Rules](https://dev.wix.com/docs/rest/crm/community/groups/rules/list-rules) endpoint.
-2. Save the rules on your server and update them if needed using the [Create or Replace Group Rules](https://dev.wix.com/docs/rest/crm/community/groups/rules/create-or-replace-all-rules) endpoint.
-3. Retrieve a list of all group IDs by calling the [List Groups](https://dev.wix.com/docs/rest/crm/community/groups/groups/list-groups) endpoint.
-4. Iterate over the list of group IDs and update the rules for each relevant group using the [Create or Replace All Rules](https://dev.wix.com/docs/rest/crm/community/groups/rules/create-or-replace-all-rules) endpoint.
+1. Get the updated group rules by calling [List Rules](https://dev.wix.com/docs/rest/crm/community/groups/rules/list-rules).
```

Comment:
Wouldn't you want to start by listening for a change event or something like that?

Reply:
```suggestion
1. Listen to [Rule Created](https://dev.wix.com/docs/rest/crm/community/moderation-rules/rule-created) and [Rule Updated](https://dev.wix.com/docs/rest/crm/community/moderation-rules/rule-updated), and or regularly get the updated group rules by calling [List Rules](https://dev.wix.com/docs/rest/crm/community/groups/rules/list-rules).
```